### ISSUE TITLE
[bug]: `drizzle-kit push` is not detecting changes correctly (Turso)

### ISSUE THREAD

**Issue by JoshStwrt** [drizzle-team/drizzle-kit-mirror#397]
I'm using Turso in local development and attempting to use `drizzle-kit push` but it's not correctly detecting schema changes. For example, it won't simply detect a new column being added to a table. It does seem to detect changes to indexes and constraints. See an example below:

**Current Schema (already pushed to the database)**

```
export let example = sqliteTable('example', {
	id: text('id').primaryKey()
})
```

**Proposed Schema Change (without unique constraint)**
```
export let example = sqliteTable('example', {
	id: text('id').primaryKey(),
	extraField: text('extra_field')
})
```

Running `drizzle-kit push` on the above returns:
```
Reading config file '/***<projectpath>***/drizzle.config.ts'

[i] No changes detected
```

But, running `drizzle-kit push` on a change such as the following:

**Proposed Schema Change (with unique constraint)**
```
export let example = sqliteTable('example', {
	id: text('id').primaryKey(),
	extraFieldAndUnique: text('extra_field').unique()
})
```

Prompts the following, noting the creation of a new unique index **but** not the new column:
```
Warning You are about to execute current statements:

CREATE UNIQUE INDEX `example_extra_field_unique` ON `example` (`extra_field`);

  No, abort
❯ Yes, I want to execute all statements
```

**drizzle.config.ts**
```
export default defineConfig({
	dialect: 'sqlite',
	driver: 'turso',
	dbCredentials: {
		url: 'http://localhost:8080'
	},

	schema: './src/lib/server/database/schema/*.ts',
	strict: true,
	verbose: true
})
```

**Versions**
```
"drizzle-kit": "0.21.1",
"drizzle-orm": "0.30.10"
```

**Comment by arxkdev** [comment]
Same problem happening to me aswell

**Comment by codewithcheese** [comment]
Me too, tracking this

**Comment by VaggelisKa** [comment]
Same for me. Good to know that it's being fixed!

**Comment by varundeva** [comment]
The same issue facing. 
drizzle-kit generate is not showing these changes.

**Comment by AndriiSherman** [comment]
fixed locally, will include in one if next releases

**Comment by AndriiSherman** [comment]
should be fixed in `drizzle-kit@0.21.2`. Feel free to reopen if it is not fixed for you!

**Comment by VaggelisKa** [comment]
It's working for me @AndriiSherman :)
